src/main/java/com/movie_collection/gui/controllers/BaseController.java

```diff
@@ -256,6 +267,38 @@ private void onActionAddCategory(ActionEvent actionEvent) throws IOException {
         actionEvent.consume();
     }
 
+    private void filterBar() {
+        searchMovies.textProperty().addListener((obs, oldValue, newValue) -> {
+            TableView tableView = (TableView) getStage().getScene().lookup("#moviesTable");
+            if(tableView != null){
+                movieModel.searchMovies(newValue);
+                tableView.refresh();
+            } else {
+                System.out.println("The table view could not be found");
+
+            }
+        });
+
+
+        //System.out.println(movieModel.getAllMovies());
```

Comment:
again why ?

Reply:
I dont know you left it in there I assumed you will need it for later

but removed in the new commit